openapi-to-har.js

```diff
@@ -103,46 +118,38 @@ const getPayload = function (openApi, path, method) {
     );
   }
 
+  const payloads = [];
   if (
-     openApi.paths[path][method].requestBody &&
-     openApi.paths[path][method].requestBody.content
+    openApi.paths[path][method].requestBody &&
+    openApi.paths[path][method].requestBody.content
   ) {
-    if (openApi.paths[path][method].requestBody.content['application/json'] &&
-      openApi.paths[path][method].requestBody.content['application/json'].schema
-    ) {
-      const sample = OpenAPISampler.sample(
-        openApi.paths[path][method].requestBody.content['application/json']
-          .schema,
-        { skipReadOnly: true },
-        openApi
-      );
-      return {
-        mimeType: 'application/json',
-        text: JSON.stringify(sample)
-      };
-    }
-    
-    if (openApi.paths[path][method].requestBody.content['multipart/form-data'] &&
-      openApi.paths[path][method].requestBody.content['multipart/form-data'].schema
-    ) {
-      const sample = OpenAPISampler.sample(
-        openApi.paths[path][method].requestBody.content['multipart/form-data'].schema,
-        {skipReadOnly: true},
-        openApi
-      );
-    
-      if (sample === undefined) return null;
-    
-      const params = [];
-      Object.keys(sample).forEach(key => params.push({'name': key, 'value': sample[key]}));
-    
-      return {
-        mimeType: 'multipart/form-data',
-        params: params,
-      };
-    }
-  }
-  return null;
+		['application/json', 'multipart/form-data'].forEach((type) => {
+		  const content = openApi.paths[path][method].requestBody.content[type]
+		  if (content && content.schema) {
+		    const sample = OpenAPISampler.sample(
+		      content.schema,
+		      { skipReadOnly: true },
+		      openApi
+		    );
+		    if (type === 'application/json') {
+		      payloads.push({
+		        mimeType: type,
+		        text: JSON.stringify(sample)
+		      });
+		    } else if (type === 'multipart/form-data') {
+		      let params = [];
```

Comment:
An alternative to this code is:

```javascript
if (sample !== undefined) {
  const params = Object.keys(sample).reduce((acc,key)=>acc.concat([{'name': key, 'value': sample[key]}]),[]);
  payloads.push({
   mimeType: type,
   params: params
  });
}
```

Reply:
If you don't use the alernative I would move `let params = []` inside the if statement and update `let` to `const`